kroxylicious/src/main/java/io/kroxylicious/proxy/internal/filter/FilterContributorManager.java

```diff
@@ -51,4 +61,13 @@ public Filter getFilter(String shortName, BaseConfig filterConfig) {
 
         throw new IllegalArgumentException("No filter found for name '" + shortName + "'");
     }
+
+    public boolean requiresConfig(String shortName) {
+        Iterator<FilterContributor> it = contributors.get();
+        while (it.hasNext()) {
+            FilterContributor contributor = it.next();
+            return contributor.requiresConfig(shortName);
```

Comment:
Because it's specced as a primitive boolean we can't differentiate between Contributors that don't support that shortname. It will get a result out of the firstc contributor it checks.

Reply:
AH crap, thanks. I. had meant to go double check that one before opening the PR thanks.